src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -124,6 +191,23 @@ public static void RemoveReadOnlyThrowsWhenFound()
             }
         }
 
+        /* Placeholder information for these tests until they can be written to run reliably.
+         * Currently such tests would create physical files (Unix) and\or certificates (Windows)
+         * which can collide with other running tests that use the same cert, or from a
+         * test suite running more than once at the same time on the same machine.
+         * Ideally, we have lower testing hooks or use Microsoft Fakes Framework to redirect
+         * and encapsulate the actual storage logic so it can be tested.
+         * See issue https://github.com/dotnet/corefx/issues/12833
```

Comment:
I tweaked the comment and added a reference to that 12223 issue. I do think however that in addition to having a stable environment we should look at the Fakes framework to simulate exceptions in AddRange\RemoveRange and perhaps extrapolate the storage in a simple List so we can test the algorithm without the storage overheap\concerns.